README.md

```diff
@@ -63,9 +63,13 @@ refresh your browser.
 Previewing Your Changes Online
 ---
 
-We have a special script to use to publish your changes using the `gh-pages`
-branch of your fork of ev3dev.github.io. If you have SSH setup for your GitHub
-account, simply run`./publish <gh-user>` where `<gh-user>` is your actual GitHub
+We have a special ruby script to use to publish your changes using the `gh-pages`
+branch of your fork of ev3dev.github.io. 
+
+It requires you to install Ruby (see previous section), but Jekyll is not needed.
+
+If you have SSH setup for your GitHub
```

Comment:
You don't actually have to have SSH setup. The default is to use https and ask for a password. I think if you run `./publish.rb <gh-user> --ssh` then it will use SSH instead.

Reply:
At least, that is how it is supposed to work.